buildkite/scripts/build-artifact.sh

```diff
@@ -22,6 +22,7 @@ dune build "--profile=${DUNE_PROFILE}" \
   ${MAINNET_TARGETS} \
   src/app/logproc/logproc.exe \
   src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe \
+  src/app/kvdb_tool/kvdb_tool.exe \
```

Comment:
My initial idea was to use some light-weight language like `Go` or `Python` to write this tool, specifically because anything in Ocaml is hard to setup on user's machine.

Like, if we expect users to use our docker, then the question arises of how do they know the tool in the Docker is the right one?

Reply:
George informed me about investigations into the rocksdb `ldb` tool. In short, we could use it instead of `mina internal kvdb eq` very easily. this tool seems available and packaged in the few distros I looked, but the slight differences in packaging across systems makes me hesitant to use it in this script instead of the guaranteed-reliable `mina internal`.

Reply:
What are the differences you refer to? Could they be mitigated by checking the version of `ldb`?

Reply:
Differences in how it's installed -- it has a different package name on every system

Reply:
@emberian I don't see it a problem for the script. We shouldn't be installing it in the script, only check that it exists in PATH and the version is correct.

And have README for users with recipes for different distributions on how to install the RocksDB toolkit.

Reply:
is it needful? what is the benefit?

Reply:
Less code to maintain mostly